### DISCUSSION TITLE
Migrating from Gitlab to GitHub

### DISCUSSION THREAD

**Discussion by stnguyen90** [codemagic-ci-cd/codemagic-docs#1734] [Q&A] [answered]
Hello,

I recently moved from GitLab to GitHub, but I can't seem to figure out how to update my app on Codemagic to connect to GitHub instead. I'd really like to retain my build numbers and distribution setup. I found some discussions where people have posted their app IDs and got help migrating stuff over. Would I be able to get some help too?

Old App:
https://codemagic.io/app/5e6b08bfd5212a1fa91abb10

New App:
https://codemagic.io/app/64798b300f88a314b769d563

**Comment by rafeysup93** [comment]
Hello,

I will certainly assist you with this. May I please have your Codemagic team ID for verification?

**Reply by stnguyen90** [reply]
@rafeysup93 thank you! Is it this?

https://codemagic.io/teams/5d706f37f7bab700114ea7f5

**Reply by stnguyen90** [reply]
@rafeysup93, is there anything else you need from me for this?

**Reply by rafeysup93** [reply, marked as answer]
Your app migration has been completed successfully.
